eventmesh-dashboard-console/src/test/java/org/apache/eventmesh/dashboard/console/unit/groupmember/testGroupMemberMapper.java

```diff
@@ -0,0 +1,141 @@
+            groupMemberMapper.addGroupMember(groupMemberEntity);
+            groupMemberEntities.add(groupMemberEntity);
+        }
+        return groupMemberEntities;
+    }
+
+    public List<GroupMemberEntity> getRemovedTimeList(String topicName, String groupName) {
+        GroupMemberEntity groupMemberEntity = new GroupMemberEntity();
+        groupMemberEntity.setTopicName(topicName);
+        groupMemberEntity.setGroupName(groupName);
+        List<GroupMemberEntity> groupEntities = groupMemberMapper.selectAllMemberByDynamic(groupMemberEntity);
+        for (GroupMemberEntity groupEntity1 : groupEntities) {
+            groupEntity1.setCreateTime(null);
+            groupEntity1.setUpdateTime(null);
+        }
+        return groupEntities;
+    }
+
+    @Test
+    public void test_addGroupMember() {
+        List<GroupMemberEntity> add1 = this.insertGroupData("add1", "groupMember");
+        Assert.assertEquals(add1, this.getRemovedTimeList("add1", "groupMember"));
+    }
+
+    @Test
+    public void test_getGroupMemberByClusterId() {
+        List<GroupMemberEntity> add1 = this.insertGroupData("getByCluster", "groupMember");
+        GroupMemberEntity groupMemberEntity = new GroupMemberEntity();
+        groupMemberEntity.setClusterId(add1.get(1).getClusterId());
+        List<GroupMemberEntity> groupByClusterId = groupMemberMapper.getGroupByClusterId(groupMemberEntity);
+        GroupMemberEntity groupMemberEntity1 = groupByClusterId.get(0);
+        groupMemberEntity1.setCreateTime(null);
+        groupMemberEntity1.setUpdateTime(null);
+        Assert.assertEquals(1, groupByClusterId.size());
+        Assert.assertEquals(add1.get(1), groupMemberEntity1);
+    }
+
+    @Test
+    public void test_deleteGroupMemberById() {
+        List<GroupMemberEntity> add1 = this.insertGroupData("getById", "groupMember");
+        GroupMemberEntity groupMemberEntity = new GroupMemberEntity();
+        groupMemberEntity.setId(add1.get(2).getId());
+        GroupMemberEntity groupMemberEntity1 = groupMemberMapper.selectGroupMemberById(groupMemberEntity);
+        groupMemberEntity1.setUpdateTime(null);
+        groupMemberEntity1.setCreateTime(null);
+        Assert.assertEquals(groupMemberEntity1, add1.get(2));
+    }
+
+    @Test
+    public void test_updateGroupMemberById() {
+        List<GroupMemberEntity> add1 = this.insertGroupData("updateById", "groupMember");
+        GroupMemberEntity groupMemberEntity = new GroupMemberEntity();
+        add1.get(1).setState("fail1");
+        groupMemberEntity.setState("fail1");
+        groupMemberEntity.setId(add1.get(1).getId());
+        groupMemberMapper.updateGroupMember(groupMemberEntity);
+        GroupMemberEntity groupMemberEntity1 = groupMemberMapper.selectGroupMemberById(add1.get(1));
+        groupMemberEntity1.setUpdateTime(null);
+        groupMemberEntity1.setCreateTime(null);
```

Comment:
Why `updateTime` and `createTime` should be `null`?

Reply:
This is because the expected results are created and updated at the same time as the actual database. Moreover, these two fields are set by default when creating and updating, so there is no need to check them. So in order to set them up

Reply:
How about just not calling its setter?